.github/workflows/ci.yml

```diff
@@ -257,7 +281,8 @@ jobs:
             [[ "$GITHUB_REPOSITORY" =~ "boost-ci" ]] || cp -r boost-ci-cloned/ci .
             rm -rf boost-ci-cloned
       - name: Setup Boost
-        env: {B2_DONT_BOOTSTRAP: 1}
+        env:
```

Comment:
Why this change? This single-line syntax for a single variable is OK, IMO

Reply:
yamllint likes it the other way; both are valid.